databricks/koalas/tests/test_series.py

```diff
@@ -31,10 +31,21 @@
 from databricks.koalas.testing.utils import ReusedSQLTestCase, SQLTestUtils
 from databricks.koalas.exceptions import PandasNotImplementedError
 from databricks.koalas.missing.series import _MissingPandasLikeSeries
+from databricks.koalas.config import set_option, reset_option
 
 
 class SeriesTest(ReusedSQLTestCase, SQLTestUtils):
 
+    @classmethod
+    def setUpClass(cls):
+        super(SeriesTest, cls).setUpClass()
+        set_option("compute.ops_on_diff_frames", True)
+
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
@itholic disable this. `compute.ops_on_diff_frames` is disabled by default because it costs a lot. We should move the test cases into `OpsOnDiffFramesEnabledTest`